.gitignore

```diff
@@ -71,3 +71,7 @@ target/
 
 # Finder
 .DS_Store
+
+# Misc
+testoutputs/
```

Comment:
Was this something for you in particular? If so, I'd be careful about adding it here since `.gitignore` could grow quite a bit depending on what folks are doing. 

Instead, try using the the unit tests! You could even update some of the "cleanup" fixtures to remove "output" on the _start_ of tests rather than the end so that you can inspect the test artifacts. If you'd like some suggestions or help with this, let me know!

Reply:
yes, actually that would be helpful! but i was planning to remove that stuff before merging it in, it's just for ease of directory access right now